feat: Add more query filter options for SLA HTML and JSON API (#158)
megaease/easeprobe: README.md

  The EaseProbe would listen on the `0.0.0.0:8181` port by default. And you can access the Live SLA report by the following URL:

>   - HTML: `http://localhost:8181/` ( the query option `?refresh=30s` refreshes the page every 30 seconds))

Reviewer: ? why did we move the refresh back here?

Author: this is only for the HTML page, not for the JSON API. so...

Author: You can make any change as you like @proditis , I just make the change quickly without thinking better idea...

Reviewer: how about these?

>   - HTML: `http://localhost:8181/` ( the query option `?refresh=30s` refreshes the page every 30 seconds))

Reviewer: ```suggestion
  - HTML: `http://localhost:8181/`
```

  - JSON: `http://localhost:8181/api/v1/sla`

You can use the following URL query options for both HTML and JSON:

>   - `name`: filter the probers with the name (ex, `?name=probe1` list the probers which name containing `probe1`)

Reviewer: ```suggestion
  - `name`: filter the probers that contain the value of name (ex, `?name=probe1` list the probers which name containing `probe1`)
```

>   - `name`: filter the probers with the name (ex, `?name=probe1` list the probers which name containing `probe1`)

Reviewer: ```suggestion
  - `refresh`: (_HTML only_) refresh the page every given seconds (ex, `?refresh=30s` refreshes the page every 30 seconds)
  - `name`: filter the probers with the name (ex, `?name=probe1` list the probers which name containing `probe1`)
```
